docs/usage/create.rst.inc

```diff
@@ -102,6 +102,8 @@ exclude foo/.bundler/gems. In borg it will not, you need to use --exclude
 '\*/.bundler/gems' to get the same effect. See ``borg help patterns`` for
 more information.
 
+In addition to using ``--exclude`` patterns, it is possible to use ``--exclude-if-present`` to specify the name of a filesystem object (e.g. a file or folder name) which, when contained within another folder, will prevent the containing folder from being backed up.  By default, the containing folder and all of its contents will be omitted from the backup.  If, however, you wish to only include the the objects specified by ``--exclude-if-present`` in your backup, and not include any other contents of the containing folder, this can be enabled through using the ``--keep-exclude-tags`` option.
```

Comment:
`you wish to only include the the objects specified by` -> `you wish to include only the objects by` (double "the")